migration/workspace-query-separation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+  <reference id="catalogFramework" interface="ddf.catalog.CatalogFramework"/>
+
+  <reference id="filterBuilder" interface="ddf.catalog.filter.FilterBuilder"/>
+
+  <reference id="xmlInputTransformer" interface="ddf.catalog.transform.InputTransformer" filter="(id=xml)"/>
+
+  <bean id="workspaceQueryMigration" class="ddf.migration.workspace.query.separation.WorkspaceQueryMigration">
+    <argument ref="catalogFramework"/>
+    <argument ref="filterBuilder"/>
+    <argument ref="xmlInputTransformer"/>
+  </bean>
+
+  <service ref="workspaceQueryMigration" interface="ddf.migration.api.DataMigratable">
+    <service-properties>
+      <entry key="id" value="workspace-query"/>
+      <entry key="name" value="Workspace Query"/>
+      <entry key="description" value="The DDF now expects queries to be separate from workspaces.
+        Formerly, queries were embedded in workspace metacards. This command will extract all
```

Comment:
We should include information that aids administrators in determining whether they need to run this. For example, since the query separation won't take effect until 2.14.0, I think it would be safe to say that  you need to run this if you're upgrading from a version older than 2.14.0.